contracts/non-fungible-tokens/andromeda-crowdfund/src/state.rs

```diff
@@ -156,3 +180,57 @@ pub(crate) fn set_tier_orders(
     }
     Ok(())
 }
+
+pub(crate) fn get_user_orders(
```

Comment:
Can we add some unit-tests for this and the below functions?

Reply:
Test case added for those functions